config/sys.config

```diff
@@ -0,0 +1,17 @@
+[
+    {sasl, [
+        {sasl_error_logger, {file, "logs/sasl.log"}}
+    ]},
+    
+    {kernel, [
+        {logger, [
+            {handler, default, logger_std_h,
+                #{formatter => {logger_formatter, #{single_line => true}}}}
+        ]}
+    ]},
+    
+    {lhttpc, [
+        {domains_to_cache, ["dynamodb.us-west-2.amazonaws.com"]},
+        {dns_cache_opts, {millisecond, 1000}}
+    ]}
+].
```

Comment:
why is `sys.config` needed on a lib?

Reply:
It shows which options should be defined to use DNS caching feature. Well, I can add some readme instead

Reply:
`sasl` and `kernel`, in this context, do not need to "be defined to use DNS caching feature". The most usual path for this is to change lhttpc.app.src, inside `application.env`.

Reply:
i think adding a section at the README with some doc is the right way to go here

Reply:
> It shows which options should be defined to use DNS caching feature. Well, I can add some readme instead

ReadME it's ok in this case.